### PR TITLE
Replace master terminology in documentation

### LINKED ISSUE #2694: Replace "master" terminology in documentation
**Is your feature request related to a problem? Please describe.**
Replace non-inclusive word "master" in `*.md` documentation file.
Since new REST APIs and settings with inclusive word "cluster manager" are added by the issue https://github.com/opensearch-project/OpenSearch/issues/1549, the documentation can be updated.

A part of issue https://github.com/opensearch-project/OpenSearch/issues/472

**Describe the solution you'd like**
Replace "master" with "cluster manager" in all `*.md` documentation file.

**Describe alternatives you've considered**
none.

**Additional context**
none.

### PR DESCRIPTION
**Pull request by xuezhou25**
### Description
Replace non-inclusive word "master" in `*.md` documentation file.
 
### Issues Resolved
Close #2694
 
### Check List
- [ ] New functionality includes testing.
  - [ ] All tests pass
- [ ] New functionality has been documented.
  - [ ] New functionality has javadoc added
- [ ] Commits are signed per the DCO using --signoff 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Review by tlfeng — APPROVED** [review]
Thanks for changing the non-inclusive term. 👍

File: rest-api-spec/src/main/resources/rest-api-spec/test/README.md [hunk 1]
```diff
@@ -319,13 +319,13 @@ Stashed values can be used in property names, eg:
     - do:
         cluster.state: {}
 
-    - set: { master_node: master }
+    - set: { cluster_manager_node: master }
```
Comment (dreamer-89) [inline rest-api-spec/src/main/resources/rest-api-spec/test/README.md:322]: Do we need to replace `master` (in value) as well ?

Comment (xuezhou25) [reply rest-api-spec/src/main/resources/rest-api-spec/test/README.md]: Yes. Changed it

**Comment by tlfeng** [comment]
In build 1236:
```
REPRODUCE WITH: ./gradlew ':server:test' --tests "org.opensearch.search.DeletePitMultiNodeTests.testDeletePitWhileNodeDrop" -Dtests.seed=9E251EB37CB6899A -Dtests.security.manager=true -Dtests.jvm.argline="-XX:TieredStopAtLevel=1 -XX:ReservedCodeCacheSize=64m" -Dtests.locale=es-HN -Dtests.timezone=Australia/Lord_Howe -Druntime.java=17

org.opensearch.search.DeletePitMultiNodeTests > testDeletePitWhileNodeDrop FAILED
    java.lang.AssertionError
        at __randomizedtesting.SeedInfo.seed([9E251EB37CB6899A:6388D672E32212A0]:0)
        at org.junit.Assert.fail(Assert.java:87)
        at org.junit.Assert.assertTrue(Assert.java:42)
        at org.junit.Assert.assertFalse(Assert.java:65)
        at org.junit.Assert.assertFalse(Assert.java:75)
        at org.opensearch.search.DeletePitMultiNodeTests$1.onNodeStopped(DeletePitMultiNodeTests.java:184)
        at org.opensearch.test.InternalTestCluster$NodeAndClient.closeForRestart(InternalTestCluster.java:1028)
        at org.opensearch.test.InternalTestCluster.restartNode(InternalTestCluster.java:1931)
        at org.opensearch.test.InternalTestCluster.restartRandomDataNode(InternalTestCluster.java:1879)
        at org.opensearch.search.DeletePitMultiNodeTests.testDeletePitWhileNodeDrop(DeletePitMultiNodeTests.java:176)
```
It's not related to the code change in this PR.

**Pull request merged**